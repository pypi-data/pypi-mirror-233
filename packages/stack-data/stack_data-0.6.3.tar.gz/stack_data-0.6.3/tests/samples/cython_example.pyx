def foo():
    bar()

cdef bar():
    raise ValueError("bar!")
