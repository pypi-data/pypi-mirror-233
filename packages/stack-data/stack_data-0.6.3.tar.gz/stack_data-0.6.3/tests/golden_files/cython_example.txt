Traceback (most recent call last):
 File "cython_example.pyx", line 2, in tests.samples.cython_example.foo
       1 | def foo():
-->    2 |     bar()
       3 | 
 File "cython_example.pyx", line 5, in tests.samples.cython_example.bar
       2 |     bar()
       3 | 
       4 | cdef bar():
-->    5 |     raise ValueError("bar!")
ValueError: bar!
